src/sajari.php

```diff
@@ -0,0 +1,442 @@
+<?php
+
+namespace Sajari;
+
+require dirname(__FILE__) . '/vendor/autoload.php';
+require dirname(__FILE__) . '/proto/doc.php';
+require dirname(__FILE__) . '/proto/query.php';
+require dirname(__FILE__) . '/document.php';
+require dirname(__FILE__) . '/search.php';
+
+interface Opt
```

Comment:
Thanks @jonathaningram that's the kind of domain knowledge I need 👍  I'll check it out.